### PR TITLE
Added ec work w/ min & max test

### PR DESCRIPTION
**Pull request by mkerns1**
## Summary

- Integrated Emergency Contact information into the form structure, ensuring proper display of full names, phone numbers, and relationships.
- Fixed the issue where the Emergency Contact's first and last name appeared as "undefined undefined" during the deletion confirmation.
- Added veteranContacts data to the test cases to support emergency contact information.
- Removed unneeded hint text and updated content related to special character changes.

## Related issue(s)

- [Ticket 73277](https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/73277)

## Acceptance criteria

- [x] Emergency Contact first and last name display correctly in the deletion confirmation modal.
- [x] Updated content based on the Figma designs.
- [x] veteranContacts array is included in test data for proper validation.

### CODE REVIEW

File: src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  required: false,
+  maxItems: 2,
+  isItemIncomplete: item =>
+    !item?.fullName?.first ||
+    !item?.fullName?.last ||
+    !item?.primaryPhone ||
+    !item?.relationship,
+  text: {
+    getItemName: item => {
+      if (item && item.fullName && item.fullName.first && item.fullName.last) {
```
Comment (coope93) [inline src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js:38]: Would it be a good idea to pull some of these out to functions that could be reused? I see we have the same logic below for the deleteDescription. I know sometimes we have put stuff in a `utils/validations.js` file, although if we are only using them on this page we could probably just put them at the bottom?

Comment (coope93) [inline src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js:38]: Would this be clearer as `item?.fullName?.first && item?.fullName?.last` ?

**Review by dellerbie — CHANGES_REQUESTED** [review]
lgtm, one comment

File: src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+};
+
+const emergencyContactsPage = {
+  uiSchema: {
+    ...titleUI(
+      content['emergency-contact-title'],
+      content['emergency-contact-description'],
+    ),
+    fullName: {
+      ...fullNameUI,
```
Comment (longmd) [inline src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js:65]: You don't have to redeclare the fields. You're essentially overwriting everything for this pattern. The fullNameUI gives you the opportunity to modify the labels to append a string--in this case "Emergency contact's". Review the `spouseFullName` or `dependentFullName`. Here is the pattern for spouse:

```
fullNameUI(title => `${content['household-spouse-name-prefix']} ${title}`),
```

File: src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js (hunk 1 above)
Comment (longmd) [reply src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js:38]: To piggyback on this... there is a high chance these methods will need broken out and imported in so you can get unit test coverage.

**Review by longmd — CHANGES_REQUESTED** [review]
The only other thing I have is to ensure you have unit test coverage for everything possible. Please run the test coverage method (`yarn test:coverage-app ezr`) and check your numbers regarding this work.

**Comment by mkerns1** [comment]
These changes are included in the refactored PR here: https://github.com/department-of-veterans-affairs/vets-website/pull/32235

**Pull request closed without merging**